PyStemmusScope/save.py

```diff
@@ -250,13 +230,14 @@ def _update_dataset_attrs_dims(dataset: xr.Dataset, forcing_dict: Dict) -> xr.Da
 
     return dataset
 
-
-def to_netcdf(config_file: str, cf_filename: str) -> str:
+#pylint: disable=too-many-locals
+def to_netcdf(config_file: str, forcing_filename:str, cf_filename: str) -> str:
```

Comment:
The function `to_netcdf` should be able to save data with information in `config` which has `Location` key. Instead of passing `forcing_filename` to this function, let's use `forcing_filename= utils.get_forcing_file(config)`